tests/src/testAssembler.cpp

```diff
@@ -70,23 +79,71 @@ auto SimpleAssemblersTest() {
                                      .insertParameter(Ikarus::FEParameter::loadfactor, load)
                                      .addAffordance(Ikarus::MatrixAffordances::stiffness);
```

Comment:
also the forces affordance is missing here, this will fail if we refactor correctly

Reply:
you cann use the affordance collection